tests/unit_test/utility_test.py

```
@@ -838,6 +838,63 @@ async def test_handle_verification(self, validate_and_send_mail_mock):
+
+        await MailUtility.format_and_send_mail(
+            mail_type=mail_type, email=email, first_name=first_name
+        )
+
+        validate_and_send_mail_mock.assert_called_once_with(
+            email, expected_subject, expected_body
+        )
+
+    @pytest.mark.asyncio
+    @patch("kairon.shared.utils.MailUtility.validate_and_send_mail", autospec=True)
+    async def test_handle_verification_with_valid_first_name(self, validate_and_send_mail_mock):
```

Ruff (0.8.2)

Redefinition of unused `test_handle_verification_with_valid_first_name` from line 843

(F811)